d2ltvm.bib

```diff
@@ -80,3 +80,15 @@ @Article{	  Nath.Tomov.Dongarra.2010
   year		= {2010},
   publisher	= {SAGE Publications Sage UK: London, England}
 }
+
+@inproceedings{Ragan-Kelley.Barnes,Adams.ea.2013,
```

Comment:
bib keys are generated automatically by `bibtool -f "%3n(author).%d(year)" d2ltvm.bib -o d2ltvm.bib`

refer to https://github.com/d2l-ai/d2l-en/blob/master/STYLE_GUIDE.md#references-1 for details